lms/static/js/jquery.gradebook.js

```diff
@@ -27,49 +21,38 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		var targetLeft = clamp(tableOrigin + offset, maxScroll, 0);
```

Comment:
I'm not familiar with this code and there are zero comments, so can you please explain why you're making this change? It seems like a potentially big one.

As an aside, some comments in this code would be *super* helpful.